tests/integration/component.go

```diff
@@ -185,12 +185,6 @@ func componentTests(args ...string) {
 
 		})
 
-		It("should create the component from the branch ref when provided", func() {
```

Comment:
We already have one test to create component from the branch ref https://github.com/openshift/odo/blob/eb3fee4e6f7c12fcc19418b8556c6da09ad4a628/tests/integration/component.go#L85-L102 so, just removing this Ruby image test.